frontend/src/data/nsf313_30/query.spec.ts

```diff
@@ -0,0 +1,145 @@
+import { describe, expect, test } from '@jest/globals';
+import { Query_nsf313_30 } from './query'
+import { Query } from '../queries/query'
+import { Categories } from './categories';
+
+const demo = new Query_nsf313_30()
+
+const black = new Query('race', 'Black')
+const four_year = new Query('collegeOrUniversityLevel',
+    '4year',)
+
+const private_enrollment = new Query('collegeOrGraduateSchoolEnrollment', 'EnrolledInPrivateCollegeOrGraduateSchool')
+const female = new Query('gender', 'Female')
+
+const all_genders = Categories['gender']
+
+const all_enrollment_types = Categories['collegeOrGraduateSchoolEnrollment']
+
+const all_university_levels = Categories['collegeOrUniversityLevel']
+
+describe('demo query', () => {
+
+    test('test race alone', () => {
```

Comment:
not a blocker- It could help to write `describe(`test category dependency`, ...` so the tests are clearly testing the category dependencies.

Reply:
done